taf/developer_tool.py

```diff
@@ -40,21 +40,25 @@ def add_target_repos(repo_path, targets_directory, namespace=''):
     namespace:
       Namespace used to form the full name of the target repositories. E.g. some_namespace/law-xml
   """
-  auth_repo_targets_dir = os.path.join(repo_path, TARGETS_DIRECTORY_NAME)
+  repo_path = Path(repo_path).resolve()
+  targets_directory = Path(targets_directory).resolve()
+  if namespace is None:
+    namespace = targets_directory.name
+  auth_repo_targets_dir = repo_path / TARGETS_DIRECTORY_NAME
```

Comment:
Can namespace ever be `None` now? If not, you can change lines 47-51. with:

```python
auth_repo_targets_dir = repo_path / TARGETS_DIRECTORY_NAME / namespace
auth_repo_targets_dir.mkdir(parents=True, exist_ok=True)
```

Reply:
That if is actually intended to check if `namespace` is an empty string. That used to be the default value, making it possible to not specify the namespace at all.